src/include/cell_inline.h

```diff
@@ -696,6 +705,8 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
     copy.len = 0; /* [-Wconditional-uninitialized] */
     copy.v = 0;   /* [-Wconditional-uninitialized] */
 
+    ps = unpack_addr == NULL ? NULL : &unpack_addr->ps;
```

Comment:
We can move this to the else section in the following code. ps is not used in if (unpack_addr == NULL) case.

Reply:
Yeah I get what you mean but GCC was raising some compiler warnings when I had it in this section only earlier.